### ISSUE TITLE
 Does the configuration of the `etcd path` parameter not support in minio federation mode?

### ISSUE THREAD

**Issue by Jacky2** [minio/minio#8433]
## Current Behavior
When CoreDNS is configured with the etcd plugin, I changed the path default configuration`path /skydns` configuration value to `path /coredns`. However, the etcd parameter configuration in the minio Federation mode does not support the `path` configuration item and is saved to the default value of `path /skydns`, making the CoreDNS domain name not valid.

```bash
cat  Corefile
```

```conf
.:53 {
    etcd {  
        stubzones 
        #path /skydns
        path /coredns 
        endpoint http://10.1.20.21:2379 http://10.1.20.22:2379 http://10.1.20.23:2379 
        upstream 8.8.8.8:53 8.8.4.4:53 /etc/resolv.conf 
        fallthrough
    }
    prometheus
    cache 160
    loadbalance
    forward . 8.8.8.8:53 8.8.4.4:53 /etc/resolv.conf
    log 
}
```

Can subsequent support for the configuration of the `etcd path` parameter in minio federation mode?

## Environment
* Version used (`minio/minio:RELEASE.2019-09-18T21-55-05Z`):

**Comment by harshavardhana** [comment]
We don't support custom paths @Jacky2 - we can support it in future what is the requirement here?

**Comment by Jacky2** [comment]
@harshavardhana Only the configurable `path` parameter is required. Since the existing CoreDNS has configured the path parameter to be`path /coredns`, it is more difficult to change back to `path /skydns`. So temporarily give up using the CoreDNS component in minio federation mode.

**Comment by harshavardhana** [comment]
This is being fixed in PR #8439

**Comment by harshavardhana** [comment]
Closing as PR #8439 is merged
